.github/workflows/docker-pr-build.yml

```
@@ -77,13 +75,38 @@ jobs:
         working-directory: ./server
         run: |
           npm run eslint
+      - name: 🔐 Dependencies security audit
+        working-directory: ./server
+        run: npm audit --production --audit-level=critical || true
+  test-server:
+    name: Server test
+    runs-on: ubuntu-22.04
+    steps:
+      - name: ⬇️ Checkout Gladys code
+        uses: actions/checkout@v3
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)